doc/schema/libdoc.02.xsd

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="keywordspec" type="KeywordSpec" />
+  <xs:complexType name="KeywordSpec">
+    <xs:sequence>
+      <xs:element name="version" type="xs:string" />
+      <xs:element name="scope" type="LibraryScope" minOccurs="0" />
+      <xs:element name="namedargs" type="SupportNamedArgs" minOccurs="0" />
```

Comment:
This comment from by original schema should be added here before the above two lines:

``` 
<!-- Both "scope" and "namedargs" elements are deprecated.
      Use "scope" and "namedargs" attributes instead. -->
```

Otherwise it's impossible to know why we have "scope" and "namedargs" elements and attributes and which one to use.